examples/angular-cli/package.json

```diff
@@ -57,6 +57,6 @@
     "jest-preset-angular": "^6.0.1",
     "protractor": "~5.4.2",
     "ts-node": "~8.0.2",
-    "typescript": "^3.3.1"
+    "typescript": "^3.3.3333"
```

Comment:
this version exists and should be used? 😮

Reply:
There have been 3332 issues that have been patched I guess